examples/httpserver/bin/hello_world_server_secure.dart

```diff
@@ -14,10 +14,10 @@
 import 'dart:async';
 import 'dart:io';
 
+String certificateChain = 'server_chain.pem';
+String serverKey = 'server_key.pem';
+
 Future main() async {
-  var certificateChain =
-      Platform.script.resolve('server_chain.pem').toFilePath();
-  var serverKey = Platform.script.resolve('server_key.pem').toFilePath();
```

Comment:
This server script, with its use of `Platform.script`, doesn't lend itself well to testing. We don't use `Platform.script` in any other app, and it is beside the point that we are trying to illustrate here, so I dropped it.